<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryWindow</class>
 <widget class="QMainWindow" name="QueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>数据查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="top_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>23</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background-image:url(:/a/top.jpg)</string>
    </property>
    <property name="text">
     <string>数据查询     </string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>621</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="num_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择显示数据条数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="num_comboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>全部</string>
    </property>
   </widget>
   <widget class="QWidget" name="query_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>620</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QLabel" name="school_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学校名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="school_text">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>104</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="course_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>课程名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="course_text">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>104</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="teacher_label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>授课教师</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="teacher_text">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>104</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="query_btn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="queryBtn_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>621</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="enrollLess_btn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>课程参加人数最少</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enrollMore_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>课程参加人数最多</string>
     </property>
    </widget>
    <widget class="QPushButton" name="markMore_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>课程评价分数最高</string>
     </property>
    </widget>
    <widget class="QPushButton" name="markLess_btn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>课程评价分数最低</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="top.qrc"/>
 </resources>
 <connections/>
</ui>
